config/puma/production.rb

```diff
@@ -17,3 +17,5 @@
   dynflow = ::Rails.application.dynflow
   dynflow.initialize! unless dynflow.config.lazy_initialization
 end
+
+plugin :systemd
```

Comment:
Will this cause any complications if I attempt to start up production environment in a local development environment?

Reply:
As long as the gem is installed, I don't think it should. It checks whether the `NOTIFY_SOCKET` env var is present:
https://github.com/sj26/puma-plugin-systemd/blob/95cd35fa49770e14344ecab0e7800450cc196602/lib/puma/plugin/systemd.rb#L57-L60
https://github.com/sj26/puma-plugin-systemd/blob/95cd35fa49770e14344ecab0e7800450cc196602/lib/puma/plugin/systemd.rb#L182-L187